shared/chat/conversation/messages/message-popup/exploding/container.js

```diff
@@ -19,12 +20,12 @@ type OwnProps = {
 const mapStateToProps = (state: TypedState, ownProps: OwnProps) => {
   const yourMessage = ownProps.message.author === state.config.username
   const meta = Constants.getMeta(state, ownProps.message.conversationIDKey)
-  const canExplodeNow =
-    meta.teamType === 'adhoc' ||
-    yourMessage ||
-    TeamConstants.getCanPerform(state, meta.teamname).manageMembers
+  const canDeleteHistory =
+    meta.teamType === 'adhoc' || TeamConstants.getCanPerform(state, meta.teamname).deleteChatHistory
```

Comment:
Can perform can't be used in adhoc? We could make it right

Reply:
Everyone can do everything in adhoc conversations right?